opensquirrel/decomposer/aba_decomposer.py

```diff
@@ -0,0 +1,185 @@
+"""
+Module containing classes that inherit from the ABADecomposer class to decompose a circuit into one
+of the Pauli ABA decompositions.
+"""
```

Comment:
```suggestion
"""Module containing classes that inherit from the ABADecomposer class to decompose a circuit into one of the Pauli
ABA decompositions.
"""
```
[see](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)